src/cascadia/TerminalSettingsEditor/MainPage.cpp

```diff
@@ -60,7 +67,28 @@ namespace winrt::Microsoft::Terminal::Settings::Editor::implementation
 
         if (clickedItemContainer != NULL)

```

Comment:
while you're here -- sorry, I know, but this is the ONLY `NULL` in all of our new code (!)
```suggestion

        if (clickedItemContainer)
```